src/box/lua/load_cfg.lua

```diff
@@ -281,6 +282,15 @@ local function check_replicaset_uuid()
     end
 end
 
+local function update_default_txn_isolation()
+    local level = box.cfg.default_txn_isolation
+    level = box.internal.normalize_txn_isolation(level)
+    local ffi = require('ffi')
+    if ffi.C.box_txn_set_default_isolation_level(level) == -1 then
```

Comment:
Not so sure this should be done via FFI: I think the code pattern for adding new options like in a76eb6ef is better — i.e., add a handle in src/box/lua/cfg.cc.